tests/integration/test_manifest_generation.py

```diff
@@ -543,10 +543,683 @@ def test_single_manifest_generation_google_sheet(
         assert sheet2["G1"].value is None
 
         # AND a copy of the Excel file is saved to the test directory for manual verification
-        if manual_test_verification_enabled:
-            workbook.save(
-                os.path.join(
-                    manual_test_verification_path,
-                    "TestManifestGeneration_test_single_manifest_generation_google_sheet.xlsx",
-                )
+        workbook.save(
+            os.path.join(
+                manual_test_verification_path,
+                "TestManifestGeneration_test_single_manifest_generation_google_sheet_with_annotations.xlsx",
+            )
+        )
```

Comment:
I assume this was revert because of a merge conflict. Could you update this to only save the workbook when the flag is set to True

Reply:
Thanks for catching that. Updated in latest commit!